react-backend/app.js

```diff
@@ -16,6 +16,7 @@ const activities = require('./routes/activities');
 const checkCB = require('./routes/cbauthentication/checkCB');
 const registerCB = require('./routes/cbauthentication/registerCB');
 const checkCBlogin = require('./routes/cbauthentication/checkCBlogin');
+const checkCBemail = require('./routes/cbauthentication/checkCBemail');
```

Comment:
This has now been changed to `CBPasswordResetInstigator.js` so needs to be changes here